src/new/form.ts

```diff
@@ -86,25 +86,23 @@ const enum PackageManager {
 	pnpm = "pnpm"
 }
 
-const projectForm = async (projectName: string, args: any[]): Promise<void> => {
-	let answer: any;
+type TemplateKeys = keyof typeof Template;
+
+type ProjectFormArgs = [
+	PackageManager,
+	TemplateKeys,
+	string,
+];
+
+const projectForm = async (projectName: string, args: ProjectFormArgs): Promise<void> => {
+	let answer: {
+		name: string;
+		packageManager: string;
+		template: Template;
+		confirm: boolean;
+	};
 	const projName: string = projectName;
```

Comment:
It is not related to your changes, but thinking enough about it, this variable is useless this way, we can already use the `projectName` it self, can you add this change as well?